_employment/job-seekers/interest-profiler.md

```diff
@@ -13,34 +13,10 @@ template: 1-topic-landing
 
 You can use tools like [O&#42;Net Interest Profiler](http://www.mynextmove.org/explore/ip) to identify the kind of work you enjoy. After taking the short questionnaire, use the results (called an O&#42;Net code) in [targeted job-search engines](http://jobcenter.usa.gov/find-a-job) that may ultimately help you identify a career that fits with the kind of work you enjoy.
 
-The O&#42;NET Interest Profiler can help you find out what your interests are and how those interests relate to the world of work; you can find out what you like to do. The O&#42;NET Interest Profiler helps you decide what kinds of careers you may want to explore. The purpose of this activity is to:
-- Learn the relationship between interests and level of preparation on job choices
-- Learn the O&#42;NET site as a resource to find knowledge, skills, and abilities (KSAs) for your proposed
-career of choice
-- Use this data to conduct a self-assessment
+Similarly, the [VA CareerScope Assessment Profile](https://va.careerscope.net/gibill) provides a detailed report on interest areas such as humanitarian, mechanical, scientific, or artistic. This should be used in conjunction with your O&#42;Net results and should be shared with your career counselor. Together you may be able to more accurately target employment opportunities.
```

Comment:
Molly, please remove the second "should be" in the second sentence. So it reads, "This should be used in conjunction with your ONet results and shared with your career counselor."